get-firo/download/index.html

```diff
@@ -73,15 +73,15 @@ <h3>{% t download.qt %}</h3>
                         </div>
                     </div>
                     
-                    <!-- Electron -->
+                    <!-- Campfire -->
                     <div class="col">
                         <p class="tagline">{% t download.option2 %}</p>
                         <div class="inner">
                             <h3>{% t download.electron %}</h3>
                             <p class="wversion">{{ site.data.downloads.firo_electron_version }}</p>
-                            <p class="changelog"><a href="https://github.com/firoorg/firo-client/releases/latest" target="_blank">{% t download.changelog %}</a></p>
+                            <p class="changelog"><a href="https://github.com/firoorg/campfire/releases/latest" target="_blank">{% t download.changelog %}</a></p>

```

Comment:
I'm confused about Campfire. I see that the repo is on Firo's organization on github, but the play store and apple store are linking to cypherstack's original version.

There are 2 versions of this wallet: cypherstack's and firo's fork. It's very important to not be confusing about this and to clarify from which repository the public builds come from and will come from in the future. Right now the feeling is that the desktop releases come from Firo's repo and the mobile releases from Cypherstack, but i doubt that's the case.